core/src/components/chip/chip.tsx

```diff
@@ -1,8 +1,10 @@
-import { Component, Host, h, Event, EventEmitter, Prop } from '@stencil/core';
-import { generateUniqueId } from '../../utils/utils';
+import { Component, Host, h, Event, EventEmitter, Prop, Element } from '@stencil/core';
+import { generateUniqueId, hasSlot } from '../../utils/utils';
 
 /**
+ * @slot prefix - Slot for the prefix icon.
  * @slot label - Slot for the label text.
+ * @slot suffix- Slot for the suffix icon.
```

Comment:
```suggestion
 * @slot suffix - Slot for the suffix icon.
```

Reply:
Missing space before the dash lead to the slot docs not being rendered properly in the readme